### PR TITLE
[oc_erchef] Correct bug where incorrect module name was used for update

### PR DESCRIPTION
**Pull request by marcparadise**
DIalyzer found this one for us...

### CODE REVIEW

**Comment by marcparadise** [comment]
ping @chef/lob

**Comment by stevendanna** [comment]
@danielsdeleo might be the best reviewer for this.

File: src/oc_erchef/apps/oc_chef_authz/src/oc_chef_policy.erl [hunk 1]
```diff
@@ -181,12 +181,6 @@ record_fields(_ApiVersion) ->
 list(#oc_chef_policy{org_id = OrgId} = P, CallbackFun) ->
     CallbackFun({list_query(P), [OrgId], rows}).
 
-update(#oc_chef_policy{
-                      org_id = _OrgId,
-                      authz_id = _PolicyAuthzId,
-                      last_updated_by = _AuthzId
-                     } = Record, CallbackFun) ->
-    chef_object_default_callback:update(Record, CallbackFun).
```
Comment (marcparadise) [inline src/oc_erchef/apps/oc_chef_authz/src/oc_chef_policy.erl]: note that the module here should be chef_object_default_callbacks, not chef_object_default_callback.  Same below as well.

**Comment by danielsdeleo** [comment]
If dialyzer said it, it must be true :stuck_out_tongue_closed_eyes: 

:+1:

**Pull request merged**